playground/mocks/data/idp/idx/authenticator-verification-phone.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
This is for sms being primary and voice call being secondary option for the code to be sent by.